src/components/content-tab-assistant.tsx

```diff
@@ -204,11 +217,35 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 				data-testid="assistant-chat"
 				className={ cx(
 					'flex-1 overflow-y-auto p-8 flex flex-col-reverse',
-					! isAuthenticated && 'flex items-end'
+					isOffline && ! isAuthenticated
+						? 'flex items-center justify-center'
+						: ! isAuthenticated
+						? 'flex items-start'
+						: ''
 				) }
 			>
 				<div className="mt-auto">
-					{ isAuthenticated ? (
+					{ isOffline ? (
+						<>
+							{ isAuthenticated ? (
+								messages.length > 0 ? (
+									<>
+										<AuthenticatedView
+											messages={ messages }
+											isAssistantThinking={ isAssistantThinking }
+											updateMessage={ updateMessage }
+											path={ selectedSite.path }
+										/>
+										<OfflineModeView />
+									</>
+								) : (
+									<OfflineModeView />
+								)
+							) : (
+								<OfflineModeView />
+							) }
+						</>
```

Comment:
Seems `AuthenticatedView` component is only rendered when offline with the following condition `isAuthenticated && messages.length > 0`. I wonder if we could simplify this block with:

```suggestion
					{ isOffline ? (
						<>
							{ isAuthenticated && messages.length > 0 && (
								<AuthenticatedView
									messages={ messages }
									isAssistantThinking={ isAssistantThinking }
									updateMessage={ updateMessage }
									path={ selectedSite.path }
								/>
							) }
							<OfflineModeView />
						</>
```

Reply:
Yeah good point, let's apply that simplification. I will refactor based on your suggestion!

Reply:
Updated in https://github.com/Automattic/studio/pull/271/commits/eb125c34fdf7980a4921e94f15e19d4acd16ff52